scene.go

```diff
@@ -11,14 +11,14 @@ var scenes = make(map[string]*sceneWrapper)
 // Scene represents a screen ingame.
 // i.e.: main menu, settings, but also the game itself
 type Scene interface {
-	// Preload is called before loading resources
-	Preload()
```

Comment:
Bad idea, I'll explain why and amongst other reasons I think this PR is a bad idea.